internal/docstore/drivertest/drivertest.go

```diff
@@ -1080,8 +1086,54 @@ func testMultipleActions(t *testing.T, coll *ds.Collection) {
 	}
 }
 
+func testUnorderedActions(t *testing.T, coll *ds.Collection) {
+	if err := cleanUpTable(newDocmap, coll); err != nil {
```

Comment:
This looks very close to testUnorderedActions from #1963, but I see some differences. I think we need to wait for that PR to go in.

Reply:
Yes, removed.